src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private final Random randomColor = new Random();
+
     public String getRandomColor() {
-        return null;
+        Colors[] colors = Colors.values();
+        int randomIndex = randomColor.nextInt(colors.length);
+        return colors[randomIndex].name();
```

Comment:
The getRandomColor() method returns a string instead of the enum value. Consider changing the method return type from String to Colors.

Reply:
done